Melange.go

```diff
@@ -1,8 +1,14 @@
+// Kaffee, Milch (50/50)
+
 package main
 
 import "fmt"
 
-// Kaffee, Milch (50/50)
+func nameWithPrice(price string) string {
```

Comment:
you're missing a `:` between the price and its type here

Reply:
oh wait, that's Go

Reply:
Ah, but I totally forgot very explicit and verbose error handling. Will fix.

Reply:
if err FTW